src/game.py

```diff
@@ -1,5 +1,13 @@
+import warnings
+from datetime import datetime
 import pgn
 
+BASE_TIME = datetime(1900, 1, 1)
+
+
+def compute_delta_time(time):
+    return (time - BASE_TIME).total_seconds()
```

Comment:
Why not set BASE_TIME on the datetime of the first move?

Reply:
Sorted